packages/material/test/renderers/MaterialCategorizationLayout.test.tsx

```diff
@@ -53,7 +53,7 @@ export const initJsonFormsStore = (initState: any): Store<JsonFormsState> => {
   const s: JsonFormsState = {
     jsonforms: {
       renderers: materialRenderers,
-      fields: materialFields
+      cells: materialCells
```

Comment:
We should be able to remove part of the state now if we aren't using a table in the tests.